ui/components/app/snaps/snap-ui-radio-group/snap-ui-radio-group.tsx

```diff
@@ -61,8 +67,14 @@ export const SnapUIRadioGroup: FunctionComponent<SnapUIRadioGroupProps> = ({
             checked={value === option.value}
             onChange={() => handleChange(option.value)}
             style={{ margin: '0' }} // radio buttons have default margins that need to be stripped to ensure proper centering
+            disabled={disabled || option.disabled}
           />
           <Text
+            className={
+              disabled || option.disabled
+                ? 'snap-ui-renderer__radio-label--disabled'
+                : ''
+            }
```

Comment:
```suggestion
            className={classnames({ 'snap-ui-renderer__radio-label--disabled': disabled || option.disabled })}
```

Slightly more readable like this perhaps?